idd/Energy+.idd.in

```diff
@@ -34502,6 +34502,4630 @@ ZoneHVAC:EvaporativeCoolerUnit,
+		\type object-list
+		\object-list ScheduleNames
+	A6,	\field Minimum Supply Air Humidity Ratio Schedule Name
+		\note Values in this schedule are used as a constraint in choosing the feasible settings for supply air flow rate and outside air fraction in each operating mode. If this field is blank, no minimum is imposed.
+		\type object-list
+		\object-list ScheduleNames
+	A7,	\field Maximum Supply Air Humidity Ratio Schedule Name
+		\note Values in this schedule are used as a constraint in choosing the feasible settings for supply air flow rate and outside air fraction in each operating mode. If this field is blank, no maximum is imposed.
+		\type object-list
+		\object-list ScheduleNames
+	A8,	\field Method to Choose Controlled Inputs and Part Runtime Fraction
+		\note Select the method that will be used to choose operating mode(s), supply airflow rate(s), outside air fraction(s) and part runtime fraction(s) in each time step. AUTOMATIC = chooses controlled inputs and part runtime fraction(s) to minimize resource use within each time step while best satisfying requested sensible cooling, dehumidification and ventilation, and subject to constraints. USER DEFINED = EMS will be used to choose controlled inputs and part runtime fraction(s) in each time step. If this field is blank, default to AUTOMATIC.
+		\type choice
+		\key AUTOMATIC
+		\key USER DEFINED
+		\default AUTOMATIC
+	A9,	\field Return Air Node Name
+		\type node
+		\note Return air node for the hybrid unit must be a zone exhaust node.
+	A10,	\field Outside Air Node Name
+		\type node
+		\note Outside air node for the hybrid unit must be an outdoor air node.
+	A11,	\field Supply Air Node Name
+		\type node
+		\note Supply air node for the hybrid unit must be a zone air inlet node.
+	A12,	\field Relief Node Name
+		\type node
+		\note Relief node for the hybrid unit must be a zone exhaust node, unless flow is being balanced elsewhere.
+	N1,	\field System Maximum Supply AirFlow Rate
+		\type real
+		\minimum> 0
+		\note The value in this field represents the maximum supply air volume flow rate among all operating modes. Values of extensive variables in lookup tables are normalized by the system maximum supply air mass flow rate that was used to build performance curves. The value in this field is used to rescale the output from exenstive variables to a desired system size.
+		\units m3/s
+	N2,	\field External Static Pressure at System Maximum Supply Air Flow Rate
+		\type real
+		\minimum> 0
+		\note Input the external static pressure when the system operates at maximum supply air flow rate. Fan affinity laws are used to scale supply fan power from the values tabulated in lookup tables, to values that match the external static pressure input to this field. If this field is blank, the supply fan power is not scaled from the values tabulated in lookup tables.
+		\units Pa
+	N3,	\field Scaling Factor
```

Comment:
Generic indeed... and yet "Scaling Factor for All Extensive Variable Values" seems odd and overly specific. The field description provides clear information about the function of this field.  I am open to less generic names, but the field is intended to everything about the system, so, what better to call it?